resources/views/sports/create.blade.php

```diff
@@ -13,58 +13,69 @@
                                 @csrf
 
                                 <div class="form-group row">
-                                    <label for="first" class="col-md-4 col-form-label text-md-right">1st place:</label>
+                                    <label for="gold--{{ $sport->id }}" class="col-md-4 col-form-label text-md-right">1st place:</label>
```

Comment:
@jstarnate not sure I approve two dashes as a separator, probably array would have been a better way for the structure.

Reply:
@PovilasKorop Noted! :+1: